content/release-notes/1.20.0.md

```diff
@@ -22,24 +22,24 @@ kubernetes: "1.17, 1.18, and 1.19"
 * [SupportBundle](https://troubleshoot.sh/docs/support-bundle/collecting/) and [Preflight](https://troubleshoot.sh/docs/preflight/introduction/) specs are no longer served on unauthenticated endpoints and instead are provided to [support-bundle and preflight binaries](https://troubleshoot.sh/docs/#installation) via [secrets](/kotsadm/troubleshooting/support-bundle/).
-* Version check now correctly reports current application version and adds information about KOTS Admin Console version, Kubernetes version, app status, etc.
+* The version check endpoint now correctly reports the current application version and adds information about KOTS Admin Console version, Kubernetes version, app status, etc.
 {{</changes>}}
 
 {{<fixes>}}
-* Airgap bundles can now only be installed if the app matches the license.
+* When an airgap bundle that does not match the installed license is uploaded, show an error message that explicitly says this instead of "failed to verify bundle signature".
 * Airgap bundle uploads via web UI no longer use a lot of memory.
-* Fixed a bug that caused airgap automated (unattended, headless) installs to pull application from the internet.
+* Fixed a bug that caused airgap automated (unattended, headless) installs to pull the application from the internet.
 * Fixed a bug that could cause Admin Console instance to become unusable if application installation failed under certain circumstances.
 * Fixed a bug that caused registry template functions to return no data.
 * Fixed a bug that caused [HasLocalRegistry](reference/template-functions/config-context/#haslocalregistry) and [LocalRegistryHost](/reference/template-functions/config-context/#localregistryhost) to return incorrect information.
-* Fixed a bug that caused [GitOps](/kotsadm/gitops/single-app-workflows/) to fail to clone empty repo on initial setup.
-* Fixed a troubleshoot bug where the result of a [secret collector](https://troubleshoot.sh/docs/collect/secret/) would have missing key.
-* Fixed a troubleshoot bug where a [run collector](https://troubleshoot.sh/docs/collect/run/) would hang if Pod is in state ImagePullBackoff.
-* Fixed a troubleshoot bug that caused [ingress analyzer](https://troubleshoot.sh/docs/analyze/ingress/) to not produce a result.
-* Fixed a troubleshoot bug that caused [database version analyzer](https://troubleshoot.sh/explore?tag=database) to fail on versions that aren't strictly semver conformant.
-* Fixed a troubleshoot bug that caused [distribution analyzer](https://troubleshoot.sh/docs/analyze/distribution/) to not recognize IBM and minikube distributions.
-* Fixed reporting of "released on" date to use date of promotion to channel rather than date download.
+* Fixed a bug that caused [GitOps](/kotsadm/gitops/single-app-workflows/) to fail to clone an empty repo on initial setup.
```

Comment:
```suggestion
* Fixed a bug that caused [GitOps](/kotsadm/gitops/single-app-workflows/) to fail on initial setup if the repo is empty.
```

Reply:
Is it really when it's "empty" or "unitialized"?

Reply:
I have no idea
I know the grammar/wording was off